api/filter/network/http_connection_manager.proto

```diff
@@ -80,6 +80,24 @@ message HttpConnectionManager {
     // populate the tag name, and the header value is used to populate the tag value. The tag is
     // created if the specified header name is present in the request's headers.
     repeated string request_headers_for_tags = 2;
+
+    // Global target percentage of requests that will be force traced if the *x-client-trace-id*
+    // header is set. Percent is resolved to the nearest 1% (rounded down). Defaults to 1 (i.e.,
+    // 100%). This variable is a direct analog for the variable of the same name in the :ref:`HTTP
+    // Connection Manager <config_http_conn_man_runtime>`.
+    Percent client_enabled = 3;
+
+    // Global target percentage of requests that will be traced after all other checks have been
+    // applied (force tracing, sampling, etc.). Percent is resolved to the nearest 1% (rounded
+    // down). Defaults to 1 (i.e., 100%). This variable is a direct analog for the variable of the
+    // same name in the :ref:`HTTP Connection Manager <config_http_conn_man_runtime>`.
+    Percent global_enabled = 4;
+
+    // Global target percentage of requests that will be randomly traced. Percent is resolved to the
```

Comment:
The concept of Global is strange here. If I run a server, that server can be a standalone server, or a server in a cluster, or a zone, or a region, or many regions. What does "Global" mean here? Does my server have to talk to a global service to do the tracing?

Reply:
Nope, it's per HTTP connection manager (as I understand it). Fixed.